src/styles/blocks/container.scss

```diff
@@ -0,0 +1,10 @@
+.container {
+  margin: 0 auto;
```

Comment:
```suggestion
```
Don't use outer spacings (margins) for BEM blocks.
`container` is a block

Reply:
Ok, but it's been done according to the video instructions.

Reply:
It's just a good practice for the future